generator/docs/GenerationTypes.md

```diff
@@ -27,13 +27,30 @@ Examples:
 | `Field 1 in set [A, B, C]` | _(A, B or C in any order, repeated as needed)_ | `null` |
 
 ## Interesting
-Generate some data that matches the boundary conditions for the given set of constraints. This mode will also emit data that is required to satisfy conditions in other constraints (e.g. `IF` constraints).
+See [this document](https://github.com/ScottLogic/datahelix/wiki/Interesting-data-generation) for more details on the _interesting generation mode_.
 
-Examples:
+The values that are generated by the generator are shown in the matrix below. `granularTo` does not have any bearing on whether the field represents integer or decimal values. If no `ofType` constraint exists then interesting values from _all_ data types can be emitted.
```

Comment:
Its actually controlled - from what i can see - by the presence of `greaterThan`/`lessThan` if either of these contain an operand (value) with a decimal point (even if its 10.0) then its treated as decimal, otherwise (or if they're mussing) its treated as an integer